src/main/io/serial_msp.c

```diff
@@ -1541,6 +1549,13 @@ static bool processInCommand(void)
         featureSet(read32()); // features bitmap
         break;
 
+#ifdef BEEPER
+    case MSP_SET_BEEPER:
+        beeperOffClearAll();
```

Comment:
Mask will be invalid until set again. It's better to keep state consistent at all times (update mask using single call)